packages/css/chip.css

```diff
@@ -1,9 +1,12 @@
 @layer fds.chip {
+  /*
+  The class is unused. There is no root chip component?
   .chip {
     display: flex;
   }
+  */
```

Comment:
?

Reply:
I was thinking we'll still have a "root" chip class; `fds-chip`. This will contain the css variables and css that make up the shape of the chip, such as height, padding, background and color. The different modifiers `toggle` and `removable` will then override them, or just `fds-chip`(instead of `fds-chip--button` for `toggle`) and `fds-chip--removable`.

Reply:
I really like this. The current classes and setup confusing to me. But this is a bigger change to the chip component. 
Should we have a separate issue on this or add it to CSS discrepancies? #1952

Reply:
I added it to the CSS discrepancies issue, was thinking I could just fix it quickly later instead of trying to explain it 😅